core/domain/learner_progress_services_test.py

```diff
@@ -481,6 +520,24 @@ def test_get_all_completed_exp_ids(self):
         self.assertEqual(learner_progress_services.get_all_completed_exp_ids(
             self.user_id), [self.EXP_ID_0, self.EXP_ID_1])
 
+        # Mark EXP_ID_2 as completed and create summary list to test output.
+        learner_progress_services.mark_exploration_as_completed(
+            self.user_id, self.EXP_ID_2)
+        exploration_ids = [self.EXP_ID_0, self.EXP_ID_1, self.EXP_ID_2]
+        exploration_summaries = exp_services.get_exploration_summaries_matching_ids( # pylint: disable=line-too-long
```

Comment:
Why do we need the exploration summaries? I think the get_all_completed_exp_ids (as used above) would suffice?

Reply:
I still do need to retrieve a list of exploration summaries (not just exploration IDs) in order to test that whatever is returned from get_activity_progress legitimately filtered the original summaries. 
I definitely can, however, change this to be 
`exp_services.get_exploration_summaries_matching_ids(
learner_progress_services.get_all_completed_exp_ids(self.user_id))`